ptcg_cockatrice.xml

```diff
@@ -931,7 +931,7 @@ As long as Machamp is evolved, it must be blocked if able.</text>
 			<set picURL="https://raw.githubusercontent.com/brooks42/ptcg/main/cards/WeepinbellDFC.png" rarity="common">PTCG</set>
 			<color>G</color>
 			<type>Pokémon - Grass Poison</type>
-			<pt>3/3</pt>
+			<pt>2/2</pt>
```

Comment:
I think Bellsprout should have G: evolve instead, since weepinbel is smaller

Reply:
Agreed